bpf/http_sock.h

```diff
@@ -99,6 +101,13 @@ struct {
     __uint(max_entries, 1);
 } tcp_req_mem SEC(".maps");
 
+struct {
+    __uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+    __type(key, int);
+    __type(value, u8[(IO_VEC_MAX_LEN * 2)]);
```

Comment:
Had to make this twice as large as the max size we need, otherwise I couldn't convince the verifier the variable sized appends to the buffer are OK.